packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -355,7 +355,7 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
   },
 
   ArrayBuffer: {
-    isView: define(null, ["es.array-buffer.is-view"]),
+    isView: define("array-buffer/is-view", ["es.array-buffer.is-view"]),
```

Comment:
It's missed in the pure version.

Reply:
![image](https://github.com/babel/babel-polyfills/assets/30521560/b9814688-a80b-4598-ad96-b8806510c229)
Is it unavailable?

Reply:
It's a cap.

https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override/modules/es.array-buffer.is-view.js